omod/src/main/resources/config.xml

```diff
@@ -53,17 +53,85 @@
 	 /Required Privileges -->
 
 
-	<!-- Required Global Properties
 	<globalProperty>
-		<property>@MODULE_ID@.someProperty</property>
-		<defaultValue></defaultValue>
+		<property>@MODULE_ID@.startDrugsConceptUuid</property>
+		<defaultValue>e1d9ef28-1d5f-11e0-b929-000c29ad1d07</defaultValue>
 		<description>
-			A description of the global property goes here.  The description
-			should explain enough for administrators to understand the purpose
-			and possible values for the global property.
+			UUID for the INICIAR concept
 		</description>
 	</globalProperty>
-	/Required Global Properties -->
+
+    <globalProperty>
+        <property>@MODULE_ID@.historicalStartDateConceptUuid</property>
+        <defaultValue>e1d8f690-1d5f-11e0-b929-000c29ad1d07</defaultValue>
+        <description>
+            UUID for the DATA DE INICIO DO TARV concept

```

Comment:
Used names in Portuguese for the descriptions.

Reply:
I think in code we can stick to english except those that will appear on the UI for the user to go through. We effect multiple locales through messages.properties files in the config

Reply:
I've used names in pt to make it easier to search for the concept, no need to change the system language. This description only appears in the UI.